src/io.jl

```diff
@@ -30,9 +30,22 @@ hg_save(fname::AbstractString, h::Hypergraph) =
 Loads a hypergraph from a stream `io`. The second argument
 `T` represents type of data in the hypegraph
 
+Skips initial comments.
+
 """
 function hg_load(io::IO, T::Type{<:Real})
-    l = split(readline(io))
+    #l = split(readline(io))
+    line = readline(io)
+    if occursin(r"\"\"\".*\"\"\"", line) #single line comment
+        line = readline(io)
+    elseif startswith(line, "\"\"\"") #multiple lines comment
+        line = readline(io)
+        while !(endswith(line, "\"\"\""))
```

Comment:
you have to add an EOF check here or you might get an infinite loop on malformed input file